crates/solver/src/objective_value.rs

```diff
@@ -0,0 +1,153 @@
+use crate::settlement::{external_prices::ExternalPrices, Settlement};
+use num::BigRational;
+use number_conversions::u256_to_big_rational;
+use primitive_types::U256;
+
+pub struct Inputs {
```

Comment:
Not a huge fan of the name (I see where its coming from though, "the inputs to the objective value"). It feels a bit general.

Don't have any better ideas though.

Reply:
The motivation for making this a struct instead of a function is that I wanted better distinction of the arguments that all have the same type. Agree the name that doesn't feel great but also don't have a better one. Referring to this as `objective_value::Inputs` is ok.

Reply:
> The motivation for making this a struct instead of a function is that I wanted better distinction of the arguments that all have the same type

I like the struct! Was really just referring to the name `Inputs`. I think the PR is a great fix!

Not blocking merging in any way, just pointed it out.